### PR TITLE
Remove Bicategories from CI

### PR DESCRIPTION
**Pull request by benediktahrens**

### CODE REVIEW

**Comment by benediktahrens** [comment]
Are you saying that you would like to keep the package `Bicategories` in the CI for `TypeTheory`?

I would say:
- Current `master` does not need `Bicategories`. Hence `Bicategories` should be removed.
- If you add material to `TypeTheory` that requires `Bicategories`, you should add `Bicategories` to CI.
- If you want to merge your material into `UniMath/TypeTheory`, then `Bicategories` should be added to CI there.

**Comment by rmatthes** [comment]
My message was badly structured: for `TypeTheory`, `Bicategories` are currently not needed, but then `SubstitutionSystems` have to be taken out as well. For the main repo `UniMath`, the question was if there could be a mechanism to selectively (de)activate the bicategorical material.

File: .github/workflows/build-typetheory.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ jobs:
           pwd
           git checkout master
           git show
-          make BUILD_COQ=yes PACKAGES="Foundations MoreFoundations Combinatorics Algebra NumberSystems PAdics CategoryTheory Bicategories Ktheory Topology SubstitutionSystems" install
+          make BUILD_COQ=yes PACKAGES="Foundations MoreFoundations Combinatorics Algebra NumberSystems PAdics CategoryTheory Ktheory Topology SubstitutionSystems" install
```
Comment (benediktahrens) [inline .github/workflows/build-typetheory.yml:44]:
```suggestion
          make BUILD_COQ=yes PACKAGES="Foundations MoreFoundations Combinatorics Algebra NumberSystems PAdics CategoryTheory Ktheory Topology" install
```

**Comment by benediktahrens** [comment]
> For the main repo `UniMath`, the question was if there could be a mechanism to selectively (de)activate the bicategorical material.

Does
```
$ make PACKAGES="Foundations MoreFoundations Combinatorics Algebra NumberSystems PAdics CategoryTheory Ktheory Topology"
```
not meet your needs?

**Comment by rmatthes** [comment]
The problem I see is that one has to change file `.github/workflows/build-typetheory.yml` in the repository for that. So, the change would be permanent (unless someone thinks of reverting it). And it think this lies in the nature of the GitHub actions that treat that meta-information the same way as the contents.

**Comment by rmatthes** [comment]
This is why I wrote above "But this is another story.". The idea was just triggered by your PR. I meant the possibility of a temporary deactivation if it is believed a PR will have no effect on `Bicategories` (and `SubstitutionSystems`). Before merging the PR, one should not only "believe", but there might be many interactions during the evolution of a PR. Anyway, I would be surprised if any easy-to-use mechanism existed for that.

**Comment by benediktahrens** [comment]
I don't think this is possible - but https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/ could be of interest.

**Comment by benediktahrens** [comment]
Superseded by #227 , closing.

**Pull request closed without merging**